account-kit/rn-signer/src/signer.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from "zod";
+import {
+  BaseAlchemySigner,
+  SessionManagerParamsSchema,
+} from "@account-kit/signer";
+// eslint-disable-next-line import/extensions
+import { RNSignerClient, type RNSignerClientParams } from "./client";
+
+const RNSignerClientParamsSchema = z.custom<RNSignerClientParams>();
```

Comment:
nit: we should move this into the client.ts file since it doesn't have a schema defined and invert it so that `RNSignerClientParamsSchema` defines the scheme explicitly while `RNSignerClientParams = z.input<typeof RNSignerClientParamsSchema>`

Reply:
Very fair shout!

Reply:
Updated!